bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/BaseThingHandler.java

```diff
@@ -96,6 +101,22 @@ public void removedService(final ServiceReference reference, final Object servic
             }
         };
         thingRegistryServiceTracker.open();
+        linkRegistryServiceTracker = new ServiceTracker(this.bundleContext, ItemChannelLinkRegistry.class.getName(),
```

Comment:
Yes, the same why the thingRegistryServiceTracker isn't using DS: The whole ThingHandlers are no DS components.